### PR TITLE
[Icons] Updates social media icons

### PR DESCRIPTION
**Pull request by heyjoethomas**
Updates social media icons by removing them from their containers so they are more identifiable. Also adds an icon for Twitch.

### WHY are these changes introduced?

We received feedback that the current icons were hard to distinguish when next to each other.

### How to 🎩

🖥 [Local development instructions](https://github.com/Shopify/polaris/blob/main/README.md#local-development)
🗒 [General tophatting guidelines](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md)
📄 [Changelog guidelines](https://github.com/Shopify/polaris/blob/main/.github/CONTRIBUTING.md#changelog)

<details>
<summary>Copy-paste this code in <code>playground/Playground.tsx</code>:</summary>

```jsx
import React from 'react';
import {Page} from '../src';

export function Playground() {
  return (
    <Page title="Playground">
      {/* Add the code you want to test in here */}
    </Page>
  );
}
```

</details>

### 🎩 checklist

- [ ] Tested on [mobile](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md#cross-browser-testing)
- [ ] Tested on [multiple browsers](https://help.shopify.com/en/manual/shopify-admin/supported-browsers)
- [ ] Tested for [accessibility](https://github.com/Shopify/polaris/blob/main/documentation/Accessibility%20testing.md)
- [ ] Updated the component's `README.md` with documentation changes
- [ ] [Tophatted documentation](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting%20documentation.md) changes in the style guide

### CODE REVIEW

**Review by stuartinstrell — COMMENTED** [review]
Thanks @heyjoethomas these look great to me! Will let @Saulki review before approving for merge.

File: polaris-icons/icons/InstagramMinor.svg
Comment (Saulki) [inline polaris-icons/icons/InstagramMinor.svg:1]: Instagram icon looks much more thin/smaller when placed next to other icons. Any way to add more weight to it?
image

**Review by Saulki — DISMISSED** [review]
Instagram icon needs to be a little heavier.

**Comment by heyjoethomas** [comment]
> Instagram icon needs to be a little heavier.

@Saulki  We are using their branding directly. We can't edit or change their logo

**Comment by nattydodd** [comment]
Hey @heyjoethomas @Saulki! just curious when you expect this to be merged? We'd like to make use of these icons in Collective when ready. Thanks!

**Comment by Saulki** [comment]
> > Instagram icon needs to be a little heavier.
> 
> @Saulki We are using their branding directly. We can't edit or change their logo

@heyjoethomas I might be missing context on branding guidelines or Polaris icon guidelines, maybe even don't have context of future plans with micro icon size (my assumption why social icons being so small), so sorry if I miss on this one.

But I've played around just with the size of these icons and come to couple of ideas, let me know if there is reason they don't work and we can revert to original icons. Check it in my Figma: https://www.figma.com/file/BVJ1lEY2dEH7gEl0ypFWsP/Application-management?type=design&node-id=3749%3A49589&mode=design&t=EOsFQsre02hDZYF3-1

My rationale:
- I found out none of social icons fit to Polaris icon keylines and are much smaller than that;
- If we are restricted by branding, we can still play with the size and aim for similar visual weight, even when actual icon size is bigger than others. Just bumping it up to keylines might be enough, but we can go beyond that if it has thin strokes like Instagram.
- I've used few other minor icons that we commonly use in Collabs as a comparison, because they will be mostly used in this context, next to interactive icons (like print or message). That was my baseline for icon weight. Most common use case for social icons is having it interactive as icon button which links out to social platform. Do you have other use cases in mind maybe?

image

**Comment by heyjoethomas** [comment]
> > > Instagram icon needs to be a little heavier.
> > 
> > 
> > @Saulki We are using their branding directly. We can't edit or change their logo
> 
> @heyjoethomas I might be missing context on branding guidelines or Polaris icon guidelines, maybe even don't have context of future plans with micro icon size (my assumption why social icons being so small), so sorry if I miss on this one.
> 
> But I've played around just with the size of these icons and come to couple of ideas, let me know if there is reason they don't work and we can revert to original icons. Check it in my Figma: https://www.figma.com/file/BVJ1lEY2dEH7gEl0ypFWsP/Application-management?type=design&node-id=3749%3A49589&mode=design&t=EOsFQsre02hDZYF3-1
> 
> My rationale:
> 
> * I found out none of social icons fit to Polaris icon keylines and are much smaller than that;
> * If we are restricted by branding, we can still play with the size and aim for similar visual weight, even when actual icon size is bigger than others. Just bumping it up to keylines might be enough, but we can go beyond that if it has thin strokes like Instagram.
> * I've used few other minor icons that we commonly use in Collabs as a comparison, because they will be mostly used in this context, next to interactive icons (like print or message). That was my baseline for icon weight. Most common use case for social icons is having it interactive as icon button which links out to social platform. Do you have other use cases in mind maybe?
> 
> image

@Saulki Gotcha. I don't expect these to perfectly line up with our guidelines because they are built for that. However, I will double check them all to make sure they are as close as possible.

**Comment by michellegerrarddoyle** [comment]
@heyjoethomas do we have an idea when this will be merged please?

**Comment by heyjoethomas** [comment]
> @heyjoethomas do we have an idea when this will be merged please?

When someone approves the changes.

**Comment by michellegerrarddoyle** [comment]
@heyjoethomas look ok to me but I know you said the original request came from Saulius and Stu.

@Saulki please could you review since you gave the last round of feedback? Keen to see it merged asap as we have a dev task dependant on it.

**Review by stuartinstrell — APPROVED** [review]
Going to approve this PR so as to not block projects for Collabs and Collective. @Saulki is currently OOO - @heyjoethomas is there a way we can get this shipped asap and Saulius can raise any issues when he returns? cc @michellegerrarddoyle

**Pull request merged**